translate/tools/junitmsgfmt.py

```diff
@@ -0,0 +1,99 @@
+from __future__ import annotations
+
+import subprocess
+import xml.etree.ElementTree as etree
```

Comment:
I see you're using lxml instead of xml.etree and deepsource marked this as error.

Should i use it here too? I used it becouse it is typed. And it's safe for generating xml. But i can switch to lxml.

Would be great to add types-lxml as dependency for type-checking support.

Reply:
lxml is safer for parsing untrusted XML, that's why it's used in most places. You are using it for generating, so it does not matter much.